src/transformers/generation/logits_process.py

```diff
@@ -1333,6 +1333,8 @@ class WhisperTimeStampLogitsProcessor(LogitsProcessor):
     Whisper specific Processor. This processor can be used to force a list of tokens. The processor will set their log
     probs to `inf` so that they are sampled at their corresponding index.
 
+    See [the paper](https://arxiv.org/abs/2303.00747) for more information.
```

Comment:
The paper here would be the original whisper paper, https://arxiv.org/abs/2212.04356 :)

The timestamps returned here are the sequence-level timestamps, not the word-level timestamps of WhisperX. (As further proof, this code was added before the WhisperX paper came out, as a result of a refactor)